javascript-view-swapping/main.js

```diff
@@ -0,0 +1,26 @@
+const $tabs = document.querySelector('.tab-container');
+const $tabList = document.querySelectorAll('.tab');
+const $viewList = document.querySelectorAll('.view');
+
+$tabs.addEventListener('click', function (event) {
+
+  if (event.target.matches('.tab')) {
+    for (let i = 0; i < $tabList.length; i++) {
+      if (event.target.dataset.view === $tabList[i].dataset.view) {
```

Comment:
You don't want to check if the view is the same, you just want to check if the element is the same